src/newbank/server/model/Account.java

```diff
@@ -0,0 +1,154 @@
+package newbank.server.model;
+
+import newbank.server.database.TransactionDB;
+
+import java.util.List;
+
+public class Account {
+
+  private long accountNumber;
+  private String accountName;
+  private double balance;
+  private TransactionDB transactionDB = TransactionDB.getInstance();
+
+  private Customer customer;
```

Comment:
I have some doubts about this line. This will raise the code complexity because now Accounts have to know about the Customer. Let's see what will happen, maybe I might be wrong.

Reply:
In Customers I am tracking only this account number 
  private ArrayList<Long> accountNumbers;


This will be the key to the account. This is done so because transfer of amount is always done between account to account. We should be able to identity account by the account number itself. Maybe when we implemented something later with account number this will come handy.

Reply:
I felt this was required at some stage because sometimes money transactions require only Account number. Here we can fetch Customer from Account also. Customer and Account are to be connected both ways. So thought will have a separate DB class to handle Accounts.